jmui/jmui/open_wallet_dialog.py

```diff
@@ -0,0 +1,136 @@
+# -*- coding: utf-8 -*-
+
+################################################################################
+## Form generated from reading UI file 'open_wallet_dialog.ui'
+##
+## Created by: Qt User Interface Compiler version 5.14.2
+##
+## WARNING! All changes made in this file will be lost when recompiling UI file!
+################################################################################
+
+from PySide2.QtCore import (QCoreApplication, QDate, QDateTime, QMetaObject,
+    QObject, QPoint, QRect, QSize, QTime, QUrl, Qt)
+from PySide2.QtGui import (QBrush, QColor, QConicalGradient, QCursor, QFont,
+    QFontDatabase, QIcon, QKeySequence, QLinearGradient, QPalette, QPainter,
+    QPixmap, QRadialGradient)
```

Comment:
Unused imports should be removed.
```
$ ./test/lint/lint-python.sh 
jmui/jmui/open_wallet_dialog.py:11:1: F401 'PySide2.QtCore.QDate' imported but unused
jmui/jmui/open_wallet_dialog.py:11:1: F401 'PySide2.QtCore.QDateTime' imported but unused
jmui/jmui/open_wallet_dialog.py:11:1: F401 'PySide2.QtCore.QObject' imported but unused
jmui/jmui/open_wallet_dialog.py:11:1: F401 'PySide2.QtCore.QPoint' imported but unused
jmui/jmui/open_wallet_dialog.py:11:1: F401 'PySide2.QtCore.QRect' imported but unused
jmui/jmui/open_wallet_dialog.py:11:1: F401 'PySide2.QtCore.QTime' imported but unused
jmui/jmui/open_wallet_dialog.py:11:1: F401 'PySide2.QtCore.QUrl' imported but unused
jmui/jmui/open_wallet_dialog.py:13:1: F401 'PySide2.QtGui.QBrush' imported but unused
jmui/jmui/open_wallet_dialog.py:13:1: F401 'PySide2.QtGui.QColor' imported but unused
jmui/jmui/open_wallet_dialog.py:13:1: F401 'PySide2.QtGui.QConicalGradient' imported but unused
jmui/jmui/open_wallet_dialog.py:13:1: F401 'PySide2.QtGui.QCursor' imported but unused
jmui/jmui/open_wallet_dialog.py:13:1: F401 'PySide2.QtGui.QFont' imported but unused
jmui/jmui/open_wallet_dialog.py:13:1: F401 'PySide2.QtGui.QFontDatabase' imported but unused
jmui/jmui/open_wallet_dialog.py:13:1: F401 'PySide2.QtGui.QIcon' imported but unused
jmui/jmui/open_wallet_dialog.py:13:1: F401 'PySide2.QtGui.QKeySequence' imported but unused
jmui/jmui/open_wallet_dialog.py:13:1: F401 'PySide2.QtGui.QLinearGradient' imported but unused
jmui/jmui/open_wallet_dialog.py:13:1: F401 'PySide2.QtGui.QPalette' imported but unused
jmui/jmui/open_wallet_dialog.py:13:1: F401 'PySide2.QtGui.QPainter' imported but unused
jmui/jmui/open_wallet_dialog.py:13:1: F401 'PySide2.QtGui.QPixmap' imported but unused
jmui/jmui/open_wallet_dialog.py:13:1: F401 'PySide2.QtGui.QRadialGradient' imported but unused
```

Reply:
This is generated code, these unused import will be re-introduced back during the next generation.
How about we exclude these generated code from the linter?

Reply:
Indeed, that appears to be necessary with this workflow.